src/adic_space.lean

```diff
@@ -8,11 +8,132 @@ import for_mathlib.topology
 
 open nat function
 
-variables (R : Type) [comm_ring R] [topological_space R] [topological_ring R]  
+section comm_ring
+variables (R : Type) [comm_ring R]
+-- This section is filled in in Johan's PR
+definition is_subring {R : Type} [comm_ring R] : set R → Prop := sorry 
+definition is_integrally_closed {R : Type} [comm_ring R] : set R → Prop := sorry  
+end comm_ring
+
+section topological_ring
+
+variables (R : Type*) [comm_ring R] [topological_space R] [comm_ring R] [topological_ring R]  
+
+def nhd_zero := (nhds (0 : R)).sets
+variable {R}
+lemma nhd_zero_symmetric {V : set R} : V ∈ nhd_zero R →  (λ a, -a) '' V ∈ nhd_zero R :=
```

Comment:
This doesn't use the ring structure, right? Should we just prove it for additive topological groups?